src/app/spec.helper.ts

```diff
@@ -17,6 +17,15 @@ export function simulationWithCriteria(
   }
 }
 
+export function simulationWithCriteriaC(
```

Comment:
I think we can rename the existing `simulationWithCriteria` to follow this new naming convention.

Reply:
ill do this in a separate MR for type B visa